doc/adr/0005-wego-core-auth-strategy.md

```diff
@@ -4,7 +4,11 @@ Date: 2021-07-22
 
 ## Status
 
-Accepted
+~Accepted~
+
+Partially superseded by [ADR-0015 Current API Authentication Strategy](./0015-current-api-authentcation-stratgey.md) with regards to content around kubernetes
```

Comment:
not super clear to me what which parts of this ADR remain valid and which do not

Reply:
I think this is also a typo, should be `strategy` not `stratgey`.